### ISSUE TITLE
Bug: This plugin has no effect

### ISSUE THREAD

**Issue by how2051** [joeriddles/extended-task-lists#1]
Hi, I just installed and enabled your plugin, but it did not show the effect as expected.

No matter what I typed into the `[]`, it all turned out to be finished.
For example, I typed the example in yr README.md:
```
- [ ] Pending
- [.] In progress 
- [~] Won't do
- [x] Done
```

and it just showed this:
![图片](https://github.com/joeriddles/extended-task-lists/assets/55384174/7dc9875e-5fff-463b-a03c-36296abf4a18)

Could it be because I changed some files before?
FYI, I changed my css before:
![图片](https://github.com/joeriddles/extended-task-lists/assets/55384174/8aa22191-5875-4f2f-b4e8-7d6ed49aad13)

my current Obsidian version is v1.5.3(Installer version v1.4.13), and I'm running Obsidian on Window11

**Comment by joeriddles** [comment]
@how2051 thanks for the bug report. I’ll look into this!

**Comment by joeriddles** [comment]
@how2051 currently only reader mode is supported. You can toggle reading view on Windows using `ctrl + e`. Is reading mode enabled in your screenshot?

**Comment by how2051** [comment]
@joeriddles   

I'm not sure if this is the `reader mode` you said ( also, I pressed `ctrl + e` but nothing happened, the mode since unchanged)

![图片](https://github.com/joeriddles/extended-task-lists/assets/55384174/1b1fffd9-665c-4d3e-b4b0-cb604dbf6a11)

![图片](https://github.com/joeriddles/extended-task-lists/assets/55384174/1e91b422-9097-4750-b615-1dba78e3eeac)

**Comment by DanShui-Yu** [comment]
![image](https://github.com/joeriddles/extended-task-lists/assets/69069739/dfd7a4d9-ca55-4735-9997-03d6685f31fe)

Notice the second line. It's already in effect.

**Comment by how2051** [comment]
yeah but the `won't do` is not as expected like the README.md like this:

![图片](https://github.com/joeriddles/extended-task-lists/assets/55384174/0ca227cf-ceb1-459b-b3dd-89ef3a3be13f)
the red one

**Comment by joeriddles** [comment]
@how2051, please update to the latest release. I have tested on Windows and this is now working as expected! Thanks for reporting this bug. Cheers.

**Comment by how2051** [comment]
@joeriddles  Hi, thank you for the patience and effort for fixing this bug.
I've updated to the latest release and do a simple test, but unfortunately, it is still not working as expected.
Maybe there's something wrong with my operation ?

![effect](https://github.com/joeriddles/extended-task-lists/assets/55384174/da3979e0-31a6-49fb-a353-24a2bfccb84a)

I try it on 1.4 version of Obsidian, it doesn't work, so I updated my Obsidian to latest 1.5, still the same.
![version](https://github.com/joeriddles/extended-task-lists/assets/55384174/b3f8bc8e-30f7-4068-a2bc-9ed41d6461a7)

and here's what I type:
```
- [ ] Pending 
- [.] In progress 
- [~] Won't do 
- [x] Done
```

which is copy from your README.md
